apps/html/index.html

```diff
@@ -1457,7 +1458,79 @@ <h1>LitJsSdk_pkpEthers has ${entries.length} functions</h1>
     </script>
     
 
-    <!-- (20): TYPES -->
+    <!-- (20): PKPWALLETCONNECT -->
+    <script>
+        if(typeof LitJsSdk_pkpWalletconnect === 'undefined') {
+            console.error("LitJsSdk_pkpWalletconnect:", LitJsSdk_pkpWalletconnect);
+        }else{
+            console.warn("LitJsSdk_pkpWalletconnect:", LitJsSdk_pkpWalletconnect);
+            window.LitJsSdk_pkpWalletconnect = LitJsSdk_pkpWalletconnect;
+        }
+        window.addEventListener('load', function() {
+
+            var root = document.getElementById('root');
+            var result = document.getElementById('result');
+            var entries = Object.entries(LitJsSdk_pkpWalletconnect);
+            var lis = entries.map(([key, value]) => `
+            <li>
+                <div id="LitJsSdk_pkpWalletconnect_${key}" class="key" onClick="(async (e) => {
+                    var fn = LitJsSdk_pkpWalletconnect['${key}'];
+                    var fnType = typeof fn;
+                    console.warn('[${key}] is type of [' + fnType + ']');
+
+                    if ( fnType === 'string' ) return;
+
+                    if( fnType === 'function' ){
+                        try{
+                            console.log('params:', globalThis.params);
+
+                            var res;
+                            try{
+                                res = new fn(globalThis.params);
+                            }catch{
+                                res = await fn(globalThis.params);
```

Comment:
Is there a reason for using globalThis over window here?

Reply:
I guess no particular reason, just a standard way to access global value in different envs? this code is generated in `gen-utils.mjs` btw for both html & react, and it's the same for every repo